internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -526,58 +524,64 @@ func (s *BackendService) convertALBListener() (*template.ALBListener, error) {
 		return nil, nil
 	}
 	var rules []template.ALBListenerRule
-	// build listener rule config from primary rule config from manifest.
-	rule, err := routingRuleConfigConverter{
-		rule:         rrConfig.Main,
-		manifest:     s.manifest,
-		httpsEnabled: s.httpsEnabled,
-	}.convert()
-	if err != nil {
-		return nil, err
-	}
-	rules = append(rules, *rule)
-	hostedZoneAliases, err := convertHostedZone(rrConfig.Main.Alias, rrConfig.Main.HostedZone)
-	if err != nil {
-		return nil, err
+	var hostedZoneAliases template.AliasesForHostedZone
+	for _, routingRule := range rrConfig.RoutingRules() {
+		rule, err := routingRuleConfigConverter{
+			rule:            routingRule,
+			manifest:        s.manifest,
+			httpsEnabled:    s.httpsEnabled,
+			redirectToHTTPS: s.httpsEnabled,
```

Comment:
Oh should this be something similar to LBWS here?
```go
httpRedirect := s.httpsEnabled
if routingRule.RedirectToHTTPS != nil {
	httpRedirect = aws.BoolValue(routingRule.RedirectToHTTPS)
}
```

Reply:
Okay so looks like we made this change as a part of [http.redirect_to_https](https://github.com/aws/copilot-cli/pull/4003) feature where we did have a requirement for LBWS to disable http redirect to https  even when domains are present but at the same time CDN is enabled. The idea being we wanted CDN to terminate the TLS and everything behind it uses HTTP. There wasn't any supported use cases for backend service and adding this filed to boolean was just so we could reuse the template for both lbws/backend.